Assets/MixedRealityToolkit.Services/InputSystem/Inspectors/NearInteractionTouchableInspector.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Did you intend to add this class? It wasn't mentioned in the PR. Maybe add a comment?